pkgs/development/python-modules/napari-npe2/default.nix

```diff
@@ -13,11 +13,14 @@
 , rich
 , setuptools-scm
 , typer
+, build
+, rich
```

Comment:
```suggestion
```

Reply:
Duplicates break eval.

Reply:
When I remove these build inputs I get following error
```
       last 10 log lines:
       > buildPhase completed in 36 seconds
       > installing
       > Executing pipInstallPhase
       > /build/source/dist /build/source
       > Processing ./npe2-0.6.1-py2.py3-none-any.whl
       > Requirement already satisfied: pydantic in /nix/store/5m262qkll34vnvjln1ac54rih2b404s7-python3.10-pydantic-1.9.1/lib/python3.10/site-packages (from npe2==0.6.1) (1.9.1)
       > Requirement already satisfied: psygnal>=0.3.0 in /nix/store/0nr6c33yi5kddahr22v4iyk188jlcx49-python3.10-psygnal-0.3.5/lib/python3.10/site-packages (from npe2==0.6.1) (0.3.5)
       > ERROR: Could not find a version that satisfies the requirement build (from npe2) (from versions: none)
       > ERROR: No matching distribution found for build
```

Reply:
It is duplicated in the argument list, you're apparently removing it from propagatedBuildInputs.

Reply:
Oh that is obvious, sorry for that